client/jetpack-connect/features.tsx

```diff
@@ -2,17 +2,25 @@ import { Icon, check } from '@wordpress/icons';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 
-export const JetpackFeatures = ( { className }: { className?: string } ) => {
+export const JetpackFeatures = ( {
+	className,
+	col1Features,
+	col2Features,
+}: {
+	className?: string;
+	col1Features?: string[];
+	col2Features?: string[];
+} ) => {
 	const translate = useTranslate();
 
-	const col1Features = [
+	col1Features = col1Features || [
```

Comment:
We created this component, and are the only ones using it. I think we can remove these default features and make the props required. 🙂

Reply:
That's good to know :) I'll make the change.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/81871/commits/a15d9d388d907b8013fa8e1230243b36e97d7515